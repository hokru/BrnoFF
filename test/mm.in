**************************
gas phase input
&cntrl
imin=0, ntx=1,
ntb=0, ntc=1, ntf=1,
cut = 999
&end

