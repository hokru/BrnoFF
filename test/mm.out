
          -------------------------------------------------------
          Amber 14 SANDER                              2014
          -------------------------------------------------------

| Run on 12/02/2016 at 10:51:07

|   Executable path: sander
| Working directory: /home/kruse/Sync/Code/intermolFF/test
|          Hostname: simien

  [-O]verwriting output

File Assignments:
|  MDIN: mm.in                                                                 
| MDOUT: mm.out                                                                
|INPCRD: wat.crd                                                               
|  PARM: wat.top                                                               
|RESTRT: restrt                                                                
|  REFC: refc                                                                  
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: mden                                                                  
| MDCRD: mdcrd                                                                 
|MDINFO: mdinfo                                                                
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

**************************                                                     
gas phase input                                                                
&cntrl                                                                         
imin=0, ntx=1,                                                                 
ntb=0, ntc=1, ntf=1,                                                           
cut = 999                                                                      
&end                                                                           
                                                                               

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
|    NONPERIODIC   ntb=0 and igb=0: Setting up nonperiodic simulation
|  *** cutoff > system size, list only builds once
|Largest sphere to fit in unit cell has radius =    36.164
| New format PARM file being parsed.
| Version =    1.000 Date = 12/02/16 Time = 10:50:07
 NATOM  =      12 NTYPES =       2 NBONH =      12 MBONA  =       0
 NTHETH =       0 MTHETA =       0 NPHIH =       0 MPHIA  =       0
 NHPARM =       0 NPARM  =       0 NNB   =      16 NRES   =       4
 NBONA  =       0 NTHETA =       0 NPHIA =       0 NUMBND =       2
 NUMANG =       0 NPTRA  =       0 NATYP =       2 NPHB   =       1
 IFBOX  =       0 NMXRS  =       3 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0


|     Memory Use     Allocated
|     Real                3372
|     Hollerith             42
|     Integer            20857
|     Max Pairs             66
|     nblistReal           144
|     nblist Int       1371021
|       Total             5464 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       0, nmropt  =       0

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       1, ntpr    =      50, ntrx    =       1, ntwr    =     500
     iwrap   =       0, ntwx    =       0, ntwv    =       0, ntwe    =       0
     ioutfm  =       0, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       1, ntb     =       0, igb     =       0, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     = 999.00000, intdiel =   1.00000

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       0

Molecular dynamics:
     nstlim  =         1, nscm    =      1000, nrespa  =         1
     t       =   0.00000, dt      =   0.00100, vlimit  =  20.00000
|  INFO: Old style inpcrd file read


--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

default_name                                                                    
 begin time read from input coords =     0.000 ps

 Number of triangulated 3-point waters found:        4

     Sum of charges from parm topology file =   0.00000000
     Forcing neutrality...

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------

|  # of SOLUTE  degrees of freedom (RNDFP):      36.
|  # of SOLVENT degrees of freedom (RNDFS):       0.
|  NDFMIN =      30.     NUM_NOSHAKE =      0     CORRECTED RNDFP =      30.
|  TOTAL # of degrees of freedom (RNDF) =      30.
 ---------------------------------------------------

     eedmeth=4: Setting switch to one everywhere

 ---------------------------------------------------
| Local SIZE OF NONBOND LIST =         54
| TOTAL SIZE OF NONBOND LIST =         54

 NSTEP =        0   TIME(PS) =       0.000  TEMP(K) =     0.00  PRESS =     0.0
 Etot   =        51.2783  EKtot   =         0.0000  EPtot      =        51.2783
 BOND   =        51.3830  ANGLE   =         0.0000  DIHED      =         0.0000
 1-4 NB =         0.0000  1-4 EEL =         0.0000  VDWAALS    =        -0.0012
 EELEC  =        -0.1034  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 ------------------------------------------------------------------------------


      A V E R A G E S   O V E R       1 S T E P S


 NSTEP =        1   TIME(PS) =       0.001  TEMP(K) =     0.00  PRESS =     0.0
 Etot   =        51.2783  EKtot   =         0.0000  EPtot      =        51.2783
 BOND   =        51.3830  ANGLE   =         0.0000  DIHED      =         0.0000
 1-4 NB =         0.0000  1-4 EEL =         0.0000  VDWAALS    =        -0.0012
 EELEC  =        -0.1034  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 ------------------------------------------------------------------------------


      R M S  F L U C T U A T I O N S


 NSTEP =        1   TIME(PS) =       0.001  TEMP(K) =     0.00  PRESS =     0.0
 Etot   =         0.0000  EKtot   =         0.0000  EPtot      =         0.0000
 BOND   =         0.0000  ANGLE   =         0.0000  DIHED      =         0.0000
 1-4 NB =         0.0000  1-4 EEL =         0.0000  VDWAALS    =         0.0000
 EELEC  =         0.0000  EHBOND  =         0.0000  RESTRAINT  =         0.0000
|E(PBS) =         0.0000
 ------------------------------------------------------------------------------


--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|    Read coords time           0.00 ( 0.70% of Total)
|    Fast Water setup           0.00 ( 0.08% of Total)
|                Build the list             0.00 (94.41% of List )
|                Other                      0.00 ( 5.59% of List )
|             List time                  0.00 (75.56% of Nonbo)
|                   Short_ene time             0.00 (96.01% of Direc)
|                   Other                      0.00 ( 3.99% of Direc)
|                Direct Ewald time          0.00 (94.94% of Ewald)
|                Force Adjust               0.00 ( 0.64% of Ewald)
|                Virial junk                0.00 ( 1.27% of Ewald)
|                Start synchronizatio       0.00 ( 1.27% of Ewald)
|                Other                      0.00 ( 1.87% of Ewald)
|             Ewald time                 0.00 (23.99% of Nonbo)
|             IPS excludes               0.00 ( 0.30% of Nonbo)
|             Other                      0.00 ( 0.15% of Nonbo)
|          Nonbond force              0.00 (65.64% of Force)
|          Bond/Angle/Dihedral        0.00 ( 0.89% of Force)
|          Other                      0.00 (33.47% of Force)
|       Force time                 0.00 (60.23% of Runmd)
|       Verlet update time         0.00 (11.15% of Runmd)
|       Other                      0.00 (28.62% of Runmd)
|    Runmd Time                 0.00 ( 4.58% of Total)
|    Other                      0.03 (94.63% of Total)
| Total time                 0.04 (88.14% of ALL  )

| Number of list builds   :          1

| Highest rstack allocated:         30
| Highest istack allocated:          6

| Final Performance Info:
| -----------------------------------------------------
| Average timings for all steps:
|     Elapsed(s) =       0.00 Per Step(ms) =       1.97
|         ns/day =      43.88   seconds/ns =    1969.04
| -----------------------------------------------------

|           Job began  at 10:51:07.779  on 12/02/2016
|           Setup done at 10:51:07.828  on 12/02/2016
|           Run   done at 10:51:07.829  on 12/02/2016
|     wallclock() was called      65 times
